suite-common/wallet-types/src/transaction.ts

```diff
@@ -184,6 +184,10 @@ export interface WalletAccountTransaction extends AccountTransaction {
     symbol: NetworkSymbol;
     rates?: TimestampedRates['rates'];
     rbfParams?: RbfTransactionParams;
+    /**
+     * prepending txs have deadline (blockheigth) when they should be removed from UI
```

Comment:
nitpick - typo in comment 
```suggestion
     * prepending txs have deadline (blockheight) when they should be removed from UI
```